test/test_nested_tensor_masking.py

```diff
@@ -3,42 +3,309 @@
 import pdb
 import sys
 import torch
-import nestedtensor
+import nestedtensor as NT
```

Comment:
If people do `import numpy as np` and `import torch as th` then why do `import nestedtensor as NT`?

I don't care though, doesn't make a difference.

Reply:
good point. renamed.